_inc/lib/admin-pages/class.jetpack-admin-page.php

```diff
@@ -279,6 +279,10 @@ public static function wrap_ui( $callback, $args = array() ) {
 			? admin_url( 'admin.php?page=jetpack_about' )
 			: 'https://jetpack.com';
 
+		$jetpack_privacy_url = ( Jetpack::is_active() || Jetpack::is_development_mode() )
```

Comment:
This update to the privacy link will have no impact on the pre-connection admin page footer, since that is built in React code like the about page